viite-UI/src/model/SelectedLinkProperty.js

```diff
@@ -144,7 +146,7 @@
     var openFloating = function (linkId, id, singleLinkSelect, visibleFeatures) {
         var canIOpen = !_.isUndefined(linkId) ? !isSelectedByLinkId(linkId) : !isSelectedById(id);
         if (canIOpen) {
-            applicationModel.toggleSelectionTypeFloating();
+            applicationModel.setSelectionType('floating');
```

Comment:
Could we transfer the notions of 'floating', 'all' and 'unknown' to a variable on LinkValues in order to avoid direct use of values.